test/SIL.XForge.Identity.Tests/Controllers/Account/AccountControllerTests.cs

```diff
@@ -97,6 +97,43 @@ public async Task ForgotPassword_IncorrectUsernameOrEmail()
             Assert.IsTrue(model.EnableErrorMessage == true);
         }
 
+        [Test]
+        public async Task ForgotPassword_UpdateResetPasswordKeyAndExpirationDate()
+        {
+            var env = new TestEnvironment();
+            var model = new ForgotPasswordViewModel
+            {
+                UsernameOrEmail = "user",
+                EnableErrorMessage = false
+            };
+            IActionResult result = await env.Controller.ForgotPassword(model);
+            UserEntity user = await env.users.Query().SingleOrDefaultAsync(x => x.Username == model.UsernameOrEmail);
+            Assert.True(user.ResetPasswordKey != "jGc6Qe4i1kgM+aA4LVczTJwfHx2YuDR/", "ResetPasswordKey not saved.");
+            Assert.True(user.ResetPasswordExpirationDate != DateTime.Now, "ResetPasswordExpirationDate not saved.");
```

Comment:
You should test `ResetPasswordExpirationDate != default(DateTime)`. This will ensure that the property was set.

Reply:
We changes to default(DateTime).